app/presenters/calculator_content_item.rb

```diff
@@ -23,6 +23,7 @@ def payload
       title: calculator.title,
       base_path: base_path,
       format: 'placeholder_calculator',
+      details: {},
```

Comment:
This looks as if it's the [same problem that occurred in SmartAnswers fairly recently][1]. @tijmenb can you make sure `calculators` is setup as a "downstream project" for `govuk-content-schemas` too? Thanks.

[1]: https://github.com/alphagov/smart-answers/pull/2539